packages/shared.test/src/mock-ontology/mockMetadata.ts

```diff
@@ -15,4 +15,5 @@
  */
 
 export const MOCK_ORIGIN = "https://mock.com";
-export const MOCK_BASE_URL = `${MOCK_ORIGIN}/api/v2/ontologies/`;
+export const MOCK_BASE_URL: `${typeof MOCK_ORIGIN}/api/v2/ontologies/` =
```

Comment:
I thought we don't need to specify types on literal types for isolatedDeclarations?

Reply:
Oh wait, is it because its a template literal type?

Reply:
Yeah, its being strict here. I think stricter than necessary. I don't need the type checker to generate: 
```ts
export const MOCK_ORIGIN: "https://mock.com";
export const MOCK_BASE_URL: `${MOCK_ORIGIN}/api/v2/ontologies/`;
```